package.json

```diff
@@ -27,7 +27,18 @@
     "test:Chrome": "node ./scripts/test.js -- --browsers Chrome",
     "publish": "node ./scripts/publish.js",
     "lint:errors": "eslint \"./**/*.js\" --quiet",
-    "lint:all": "eslint \"./**/*.js\""
+    "lint:all": "eslint \"./**/*.js\"",
+    "lint:fix": "eslint \"./**/*.js\" --fix"
```

Comment:
Can we use single quote instead of escaping the double?

Reply:
No we can't  because eslint could not match the patters when it is with single quotes 
![image](https://user-images.githubusercontent.com/29014573/157828076-abce49c5-b3d1-47c3-ac2a-8916597a5940.png)